app/components/login_button_component.scss

```
@@ -0,0 +1,43 @@
+@use 'uswds-core' as *;
+
+.login-button.usa-button--big > .login-button-img {
```

* With BEM semantics, I'd expect the image to be an "element", i.e. delimited with `__`
* I think "img" is pretty readable but personally I'd prefer it unabbreviated

```suggestion
.login-button.usa-button--big > .login-button__image {
```